src/mono-runtimes/mono-runtimes.targets

```diff
@@ -45,7 +45,7 @@
     <_MonoSdksLlvmSrc>$(MSBuildThisFileDirectory)..\..\external\llvm</_MonoSdksLlvmSrc>
     <_MonoSdksMxePrefixDir>$(AndroidToolchainDirectory)</_MonoSdksMxePrefixDir>
     <_MonoSdksMxeSrc>$(MSBuildThisFileDirectory)..\..\external\mxe</_MonoSdksMxeSrc>
-    <_MonoSdksParameters>CONFIGURATION=$(_MonoSdksConfiguration) IGNORE_PROVISION_MXE=false IGNORE_PROVISION_ANDROID=true ANDROID_NDK_VERSION=r$(AndroidNdkVersion) ANDROID_SDK_VERSION_armeabi-v7a=$(AndroidNdkApiLevel_ArmV7a) ANDROID_SDK_VERSION_arm64-v8a=$(AndroidNdkApiLevel_ArmV8a) ANDROID_SDK_VERSION_x86=$(AndroidNdkApiLevel_X86) ANDROID_SDK_VERSION_x86_64=$(AndroidNdkApiLevel_X86_64) ANDROID_TOOLCHAIN_DIR=&quot;$(_MonoSdksAndroidToolchainDir)&quot; ANDROID_TOOLCHAIN_CACHE_DIR=&quot;$(_MonoSdksAndroidToolchainCacheDir)&quot; ANDROID_TOOLCHAIN_PREFIX=&quot;$(_MonoSdksAndroidToolchainPrefix)&quot; LLVM_SRC=&quot;$(_MonoSdksLlvmSrc)&quot; MXE_PREFIX_DIR=&quot;$(_MonoSdksMxePrefixDir)&quot; MXE_SRC=&quot;$(_MonoSdksMxeSrc)&quot;</_MonoSdksParameters>
+    <_MonoSdksParameters>CONFIGURATION=$(_MonoSdksConfiguration) IGNORE_PROVISION_MXE=false IGNORE_PROVISION_ANDROID=true ANDROID_CMAKE_VERSION=$(AndroidCmakeVersionPath) ANDROID_NDK_VERSION=r$(AndroidNdkVersion) ANDROID_SDK_VERSION_armeabi-v7a=$(AndroidNdkApiLevel_ArmV7a) ANDROID_SDK_VERSION_arm64-v8a=$(AndroidNdkApiLevel_ArmV8a) ANDROID_SDK_VERSION_x86=$(AndroidNdkApiLevel_X86) ANDROID_SDK_VERSION_x86_64=$(AndroidNdkApiLevel_X86_64) ANDROID_TOOLCHAIN_DIR=&quot;$(_MonoSdksAndroidToolchainDir)&quot; ANDROID_TOOLCHAIN_CACHE_DIR=&quot;$(_MonoSdksAndroidToolchainCacheDir)&quot; ANDROID_TOOLCHAIN_PREFIX=&quot;$(_MonoSdksAndroidToolchainPrefix)&quot; LLVM_SRC=&quot;$(_MonoSdksLlvmSrc)&quot; MXE_PREFIX_DIR=&quot;$(_MonoSdksMxePrefixDir)&quot; MXE_SRC=&quot;$(_MonoSdksMxeSrc)&quot;</_MonoSdksParameters>
```

Comment:
Here is where `$(AndroidCmakeVersion)` vs `$(AndroidCmakeVersionPath)` confuses me: we have:

```
ANDROID_CMAKE_VERSION=$(AndroidCmakeVersionPath)
```

If `$(AndroidCmakeVersionPath)` were truly a path, then this value should be quoted:

```
ANDROID_CMAKE_VERSION=&quot;$(AndroidCmakeVersionPath)&quot;
```

Should it be quoted?  I'm not sure.

Reply:
it *is* truly a path, a subdir in the SDK and there's no need to quote it since it will not contain spaces